src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -344,6 +349,13 @@ private FlightBooleanOperationExpression createFlightFilter(
           makePredicateFlightClass(FlightFilterOp.NOT_EQUAL, FileIngestWorkerFlight.class));
     }
 
+    // Only return recent flights. This is a performance enhancement since it causes the query to
+    // NOT do a full table scan.
+    // TODO: make this an option in the API.  Not tackled yet since I'm not sure of the value
+    topLevelBooleans.add(
+        makePredicateSubmitTime(
+            FlightFilterOp.GREATER_THAN, Instant.now().minus(DAYS_BACK_TO_QUERY, ChronoUnit.DAYS)));
```

Comment:
I think `Duration` is clearer here
```suggestion
            FlightFilterOp.GREATER_THAN, Instant.now().minus(Duration.ofDays(DAYS_BACK_TO_QUERY)));
```

Also since it's a constant I would declare the delta as a `Duration`
```
  @VisibleForTesting static final Duration TIME_BACK_TO_QUERY = Duration.ofDays(30);
```
That way we can avoid putting descriptors like `DAYS` in a name.

Reply:
Ah good call on Duration.  I'll update.  I also moved away from a constant and made it an application property